modules/virt-defining-apps-for-dr.adoc

```diff
@@ -44,8 +42,12 @@ There are several actions you can take to improve your experience and chance of
 === Protect the VM when using MTV, the {VirtProductName} web console, or a custom VM
 Because automatic labeling is not currently available, the application owner must manually label the components of the VM application when using MTV, the {VirtProductName} web console, or a custom VM.
 
-After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret`, and `ConfigMap`. Do not label virtual machine instances (VMIs) or pods
-since {VirtProductName} creates and manages these automatically.
+After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret`, and `ConfigMap`. Do not label virtual machine instances (VMIs) or pods; {VirtProductName} creates and manages these automatically.
```

Comment:
what about the new CR... I don't remember the name but it was something like "preferences"

Reply:
Are you referring to `VirtualMachineClusterPreference` and `VirtualMachinePreference`? 

@aglitke @kgoldbla do we have a complete/updated list of the VM resources that should be labeled?

Reply:
I don't think there is a complete list, it is entirely conceivable that other operators add crds that are related to the application that includes the VMs. But I think what @mulbc is talking about is `VirtualMachineInstanceTypes` and `VirtualMachinePreferences` the cluster versions of those should be able in all OCP-V installs as they are provided by RH. The customer can add their own VMIT and VMP so those should probably be labeled as well.

Reply:
Please add those CRs to the list in that case.
I would also expect a note stating that there might be more CRs in case the customer added more objects than the default VM

Reply:
How does this look? 
```
After creating the VM, apply a common label to the following resources associated with
the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`,
`Secret`, `ConfigMap`, `VirtualMachinePreference`, and `VirtualMachineInstancetype`. 
Do not label virtual machine instances (VMIs) or pods; {VirtProductName} creates and
manages these automatically.

[IMPORTANT]
====
You must apply the common label to everything in the namespace that you want to protect,
including objects that you added to the VM that are not listed here.
====
```